code/game/objects/items/storage/belt.dm

```diff
@@ -619,6 +620,37 @@
 			return FALSE
 	return TRUE
 
+/obj/item/storage/belt/gun/attackby(obj/item/I, mob/user, params)
+	if(!istype(I, /obj/item/weapon/gun/pistol) || !reload_on)
+		return ..()
+	var/obj/item/weapon/gun/pistol/gun = I
+	if(gun.current_mag)
+		return ..()
+	for(var/obj/item/ammo_magazine/mag in contents) 
+		if(!istype(gun, mag.gun_type))
+			continue
+		else
+			gun.reload(user, mag)
+			orient2hud()
+			return
+
+/obj/item/storage/belt/gun/examine(mob/user, distance, infix, suffix)
+	. = ..()
+	to_chat(user, "<span class='notice'>Control-Click to toggle \'Tactical Reload Mode\'. While on, clicking on the belt with an empty pistol will perform a reload with the ammo inside.</span>")
+	if(!reload_on)
+		to_chat(user, "<span class='notice'>Reload mode is OFF</span>")
+	if(reload_on)
+		to_chat(user, "<span class='notice'>Reload mode is ON</span>")
+
+/obj/item/storage/belt/gun/CtrlClick(mob/user)
+	if(src.loc != user)
```

Comment:
don't src.loc, just do src

Reply:
why? since ```src``` is the belt, and ```src.loc``` is the mob/tile where it is currently located. If I change it wouldn't ```src != user``` always equal true since ```/obj/item/storage/belt/gun``` doesn't ever equal ```/mob/user``` ?